Examples/tvOSMultiplePlayersTemplate/tvOSMultiplePlayersTemplate/PlayersCollectionViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  PlayersCollectionViewController.swift
+//  tvOSMultiplePlayersTemplate
+//
+//  Created by Aleksandr Pavliuk on 6/13/17.
+//
+//
+
+import UIKit
+import OneMobileSDK
+import AVFoundation
+
+class PlayersCollectionViewController: UICollectionViewController {
+    
+    var playerControllers:[SystemPlayerViewController] = []
+    var playersQuantity = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        for _ in 0..<self.playersQuantity {
+            var provider = OneSDK.Provider.default
+            provider.context.extra = ["noAds": true]
+            provider.getSDK()
+                .then { $0.getPlayer(videoID:"593967be9e45105fa1b5939a", autoplay: false) }
+                .dispatch(on: .main)
+                .onComplete(callback: handle)
+        }
+        
+    }
+    
+    func handle(result: Result<One.Player>) {
+        switch result {
+        case .value(let player):
+            
+            let systemPlayerViewController = SystemPlayerViewController()
+            systemPlayerViewController.player = player
+            systemPlayerViewController.view.frame = CGRect(x: 0, y: 0, width: 1800, height: 768)
+            
+            
+            self.playerControllers.append(systemPlayerViewController)
+            
+            let index = self.playerControllers.count - 1
```

Comment:
We already know that needed index it's index of last object in array, also call `index(of...` will take CPU resources.